### ISSUE TITLE
[bug] running Postgres tests fail

### ISSUE THREAD

**Issue by Treggats** [tempestphp/tempest-framework#364]
We now support PostgreSQL, how ever there are issues when running tests locally or in the CI.

The following error is thrown;

```shell
PDOException: SQLSTATE[55000]: Object not in prerequisite state: 7 ERROR:  lastval is not yet defined in this session
```

During a quick debugging session I found two issues.
- using backticks for table names and column names
   (which makes no sense)
- disabling / enabling the foreign keys during dropping all the tables

This needs some more debugging before we can get to a fix for this

**Comment by brendt** [comment]
Duplicate of #354
